ocaml/sdk-gen/powershell/autogen/README_51.md

```diff
@@ -82,82 +74,85 @@ Getting Started
 2.  Navigate to the extracted XenServerPowerShell directory and copy the whole
     folder XenServerPSModule into your PowerShell modules directory.
 
-    - On Windows this will normally be $env:UserProfile\Documents\WindowsPowerShell\Modules
-      for per-user configuration, or $env:windir\system32\WindowsPowerShell\v1.0\Modules for
-      system-wide configuration.
+    - On Windows this will normally be `$env:UserProfile\Documents\PowerShell\Modules`

```

Comment:
This is the README for 5.1, please restore the previous text because the paths are not the same as PS7.